src/main/resources/de/qabel/desktop/ui/about/about.fxml

```diff
@@ -1,83 +1,166 @@
-                <Label text="%thanksAllLabel"/>
-                <Button styleClass="button-about" text="%showSupporters" fx:id="thanksButton" onAction="#openThanksPopUp" />
-                <Label styleClass="headline" text="%aboutHeadline" />
-                <Separator />
-                <VBox styleClass="labeled-link">
+                <Label fx:id="appVersion" styleClass="label-version" text="%title" />
+                <Label styleClass="label-about-bold" text="%qabelQAPLLabel" />
+                <StackPane fx:id="QAPLBox" styleClass="box-button-about">
                     <children>
-                        <Label text="%imprint" />
-                        <Hyperlink text="%imprintUrl" />
+                        <Button fx:id="QAPLbutton" onAction="#openQAPLPopUp" styleClass="btn-about" text="%qapl" />
+                        <Hyperlink onAction="#openQAPLPopUp" styleClass="hyperlink-box-about" text="%qaplUrl" StackPane.alignment="BOTTOM_LEFT">
+                            <StackPane.margin>
+                                <Insets bottom="10.0" left="10.0" />
+                            </StackPane.margin>
+                        </Hyperlink>
                     </children>
-                </VBox>
-                <VBox styleClass="labeled-link">
+                </StackPane>
+                <HBox styleClass="label-picture">
                     <children>
-                        <Label text="%privacy" />
-                        <Hyperlink text="%privacyUrl" />
+                        <Label styleClass="label-qabel-letter"/>
+                        <Label text="%thanksAllLabel" styleClass="label-picture-left"/>
                     </children>
-                </VBox>
-                <VBox styleClass="labeled-link">
+                </HBox>
+                <VBox fx:id="thanksBox" styleClass="box-button-about">
                     <children>
-                        <Label text="%terms" />
-                        <Hyperlink text="%termsUrl" />
+                        <Button fx:id="thanksButton" onAction="#openThanksPopUp" styleClass="btn-about" text="%showSupporters" />
                     </children>
                 </VBox>
-                <VBox styleClass="labeled-link">
+                <HBox styleClass="label-picture">
                     <children>
-                        <Label text="%qapl" />
-                        <Hyperlink text="%qaplUrl" />
+                        <Label styleClass="label-qabel-letter"/>
+                        <Label text="%aboutHeadline" styleClass="label-picture-left"/>
                     </children>
-                </VBox>
-
-                <Separator />
-                <Label styleClass="headline" text="%thirdPartyComponents" />
-                <VBox styleClass="labeled-link">
+                </HBox>
+                <StackPane fx:id="imprintBox" styleClass="box-button-about">
                     <children>
-                        <Label text="Inno Setup"/>
```

Comment:
In Addition to the dependencies from gradle listed the license-dependency.xml, we got some more (Inno Setup, Launch4j, Google Material Design Icons, Community Icons, Source Sans Pro). They are not gradle dependencies so we can't collect them like the others, but they need to be included.